### ISSUE TITLE
how can I do transfer learning with my dataset ? 

### ISSUE THREAD

**Issue by bowin** [jfzhang95/pytorch-video-recognition#6]
Please help

**Comment by jfzhang95** [comment]
1. You can write your own dataloader to fit your dataset.
2. Change num_classes to fit your dataset's label in [here](https://github.com/jfzhang95/pytorch-video-recognition/blob/0c7e430c837d4e7f6ac76d051dda7df884da873e/train.py#L61).
